pyproject.toml

```diff
@@ -31,7 +31,8 @@ dependencies = [
     "uvicorn",
     "fastapi",
     "pydantic_settings",
-    "httpx"
+    "httpx",
+    "vrmapi @ git+https://github.com/mduffin95/vrm-api-python-client@fe16d06"
```

Comment:
I forked this repo from Victron. The original repo was not being maintained and so needed to be updated to work with a later version of python.

This dependency is pinned to a specific commit so it should be stable.

https://github.com/mduffin95/vrm-api-python-client

Reply:
Thanks, if you wanted OCF could in house that an publish it to pypi, then other people can use that nice. What do you think?

Reply:
Yeah I think that would be a good idea

Reply:
Thanks @mduffin95 , well i can definately setup a repo for this. I wonder first if we should try a PR back to the original [repo](https://github.com/victronenergy/vrm-api-python-client), see what they say

Reply:
The main change I made on my fork was to incorporate the changes from [this PR](https://github.com/victronenergy/vrm-api-python-client/pull/4) which has been open for years on the upstream repo. I therefore think it's unlikely that they will accept any new PRs.

Reply:
Should be able to update this to `ocf_vrmapi` now